common.js

```diff
@@ -13,7 +13,7 @@ const submodules = [
   'events', // Events and emitter
   'units', // Units conversion
   'network', // Network utilities
-  'id', // Kyes and identifiers
+  'id', // Keys and identifiers
```

Comment:
This should probably be done in a separate PR.

Reply:
No, just open another PR with this change and remove the commit with this change from this PR.

Reply:
@SemenchenkoVitaliy also there are some more typos in this file, you can fix them as well in this new PR.